File: reservation/reservation-api/src/main/kotlin/com/surfshop/reservation/dto/ReservationDto.kt
package com.surfshop.reservation.dto

import com.surfshop.common.ReservationId
import java.time.LocalDate

data class ReservationDto(
        val date: LocalDate,
        val id: ReservationId,
        val board: ReservedBoardDto,
        val sail: ReservedSailDto,
        val customer: CustomerDto

) {

}

File: webapp/src/app/reservation/service/dto/reservation-dto.ts
import {ReservedSailDto} from './reserved-sail-dto';
import {ReservedBoardDto} from './reserved-board-dto';
import {CustomerDto} from './customer-dto';

export interface ReservationDto {
  date: string;
  id: string;
  sail: ReservedSailDto;
  board: ReservedBoardDto;
  customer: CustomerDto;
}
